cronjobs/create_pipeline_execution.py

```diff
@@ -0,0 +1,123 @@
+#!/bin/python3
+# -*- coding: utf-8 -*-
+
+# Algorithm to bring pipeline phases in an ideal order
+
+def split_to_equal_parts(lst, n):
+        for x in range(0, len(lst), n):
+            each_chunk = lst[x: n+x]
+            if len(each_chunk) < n:
+                each_chunk = each_chunk + [None for y in range(n-len(each_chunk))]
+            yield each_chunk
+
+def contains_elem(lst_2d, elem):
+    result = False
+    for lst in lst_2d:
+        if elem in lst:
+            result = True
```

Comment:
Can be made shorter, just return True directly here and return False in the last line (no need for a result variable here, also => shorter execution)